tools/ui-components/src/image/image.tsx

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/display-name */
```

Comment:
I think we will encounter these a lot, so we might want to update the eslint config to disable these rules for the component library. I can help you with this in a follow-up PR, so don't worry.